middleman/source/docs/installation.html.erb

```diff
@@ -8,5 +8,28 @@ title: Bottr: Docs
     </div>
     <div class="content">
         <h1>Installation</h1>
+        It should only take three simple steps to get up and running with Bottr.
+
+Install Node & NPM using Homebrew
+```
+$ brew install node
+```
+
+Install the Bottr command line tool
+
+```
+npm install -g bottr
```

Comment:
This should be `bottr-cli`. I must have misreferenced in a while back. (I'll update the wiki)